generator/sources/node/tests/config.js

```diff
@@ -0,0 +1,23 @@
+ var minus2_partner_id = -2;
```

Comment:
why not call this admin_console_partber_id? this would be the more appropriate name
Same comment for the secret params as well